### PR TITLE
Added setup-feeder option to distributed cleanup script

### PR DESCRIPTION
**Pull request by craigcomstock**
This enables an administrator to setup a single feeder.

/opt/cfengine/federation/bin/distributed_cleanup.py --setup-feeder feederhostname

Ticket: ENT-11844
Changelog: title

### CODE REVIEW

File: templates/federated_reporting/distributed_cleanup.py [hunk 1]
```diff
@@ -253,6 +254,12 @@ def main():
             sys.exit(1)
 
     fr_distributed_cleanup_password = read_secret(DISTRIBUTED_CLEANUP_SECRET_PATH)
+    if args.setup_feeder:
+        email = input("Enter email for fr_distributed_cleanup accounts: ")
```
Comment (vpodzime) [inline templates/federated_reporting/distributed_cleanup.py:258]: This should handle non-interactive mode or it should fail properly (and the help output should mention it too).

Comment (craigcomstock) [reply templates/federated_reporting/distributed_cleanup.py:258]: will check.

File: templates/federated_reporting/distributed_cleanup.py [hunk 2]
```diff
@@ -286,7 +293,11 @@ def main():
             cert_path=CERT_PATH,
             hostname=feeder_hostname,
         )
-        response = feeder_api.status()
+        try:
+            response = feeder_api.status()
+        except Exception as e:
+            print("Could not connect to {}, error: {}".format(feeder_hostname,e));
```
Comment (larsewi) [inline templates/federated_reporting/distributed_cleanup.py:299]:
```suggestion
            print("Could not connect to {}, error: {}".format(feeder_hostname, e));
```

**Review by larsewi — APPROVED** [review]
🚀

**Review by vpodzime — APPROVED** [review]

**Review by larsewi — APPROVED** [review]

**Comment by craigcomstock** [comment]
cherry picks
https://github.com/cfengine/masterfiles/pull/2899
https://github.com/cfengine/masterfiles/pull/2898

**Pull request merged**